docs/api-product.md

```diff
@@ -0,0 +1,73 @@
+---
+id: api-product
+title: API products onboarding
+sidebar_label: API products
+---
+
+In this section it is presented how to onboard and start selling an API-based
+product with metered billing based on the actual usage, i.e. HTTP calls
+executed.
+
+The onboarding process starts after creating a new "Product" in the Marketplace
+Catalog, as described [here](onboarding.md#applications-catalogue).
+
+For extended integration purposes, it's possible to define a syndication
+endpoint like in [syndicated products](syndication.md) to receive events about
+the creation and termination of the related subscriptions for a specific
+product.
+
+## Authentication
+
+Secure access to the API is managed by the platform, generating an **unique
```

Comment:
a unique

http://www.writersdigest.com/online-editor/a-before-consonants-and-an-before-vowels-is-not-the-rule